.eslintrc.js

```diff
@@ -10,6 +10,9 @@ module.exports = {
   parserOptions: {
     ecmaVersion: "latest",
     sourceType: "module",
+    ecmaFeatures: {
+      jsx: true,
+    },
```

Comment:
We don't need this any more since you moved the actual frontend code out, right?

```suggestion
```

Reply:
yep leftt over will remove